### ISSUE TITLE
Token expires after a minute

### ISSUE THREAD

**Issue by andruu** [sprocketbox/laravel-jwt#2]
I'm using PHP 7.4. Here is the config that I'm using.

```
'api' => [
    'driver' => 'jwt',
    'provider' => 'users',
    'key' => env('JWT_KEY'),
    'ttl' => 'P1M',
],
```

**Comment by andruu** [comment]
The following always equates to one minute

```
$time   = Carbon::now();
$expiry = CarbonInterval::fromString('P1M');

$time->copy()->add($expiry)->timestamp
```

**Comment by andruu** [comment]
Passing `1 month` as the ttl works.

**Comment by ollieread** [comment]
@andruu I will look into this, as that doesn't seem right.

**Comment by ollieread** [comment]
@andruu This has been fixed by #5 and version [v1.1.2](https://github.com/sprocketbox/laravel-jwt/releases/tag/v1.1.2) contains this fix.
